yr/scripts/tutorialPage.js

```diff
@@ -243,6 +243,11 @@ var setupTutorial = function() {
     updateTutorialPage(tutorialIndex);
     return false;
   });
+
+  // Add show/hide hint functionality
+  $(".hintContainer").off("click").on("click", function(e) {
```

Comment:
I'm wondering if we want it to be `".hintContainer"` or `".hintContainer>button"`. With the code as is, if you attempt to select any text in the hint `<div>` it will collapse the hint.